features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -283,6 +284,20 @@ data class RumConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Sets the configuration for the slow frames listener, which is used to compute the slow frames array,
+         * slow frames ratio, and freeze ratio.
+         *
+         * Setting this property to null will disable the slow frames listener and stop the corresponding ratio computation.
+         *
+         * @param slowFrameListenerConfiguration The configuration to be used with the slow frames listener.
+         */
+        @ExperimentalRumApi
+        fun setSlowFrameListenerConfiguration(slowFrameListenerConfiguration: SlowFrameListenerConfiguration): Builder {
```

Comment:
sanity check: will it be the same API in iOS SDK? Will it be a permanent public API?